src/GafferSceneUI/CameraVisualiser.cpp

```diff
@@ -74,9 +74,6 @@ class CameraVisualiser : public ObjectVisualiser
 				return nullptr;
 			}
 
-			IECoreScene::CameraPtr fullCamera = camera->copy();
```

Comment:
Just noting that we should be able to close https://github.com/GafferHQ/gaffer/issues/276 now.